website/docs/components/app-footer/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: AppFooter
+description: The AppFooter is a footer that appears on every screen to displays supplementary information and links.
+caption: Displays supplementary information and links for the application.
+links:
+  figma: {link to the "AppFooter" page in the components Figma library}
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/app-footer
+related: ['layouts/app-frame']
+previewImage: assets/illustrations/components/app-footer.jpg
```

Comment:
[blocker] let's not forget to add this image before merging

Reply:
@andgen404 I know you saw this but just tagging you for extra visibility.